### PR TITLE
reenable interactive completer

### PR DESCRIPTION
**Pull request by williexu**
---
reenabled `interactive` completion in interactive mode as it is in help and is technically a valid command now.

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [na ] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [na ] Each command and parameter has a meaningful description.
- [na ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-interactive/azclishell/az_completer.py [hunk 1]
```diff
@@ -90,8 +88,7 @@ def _get_weight(val):
 
     completions = []
     for comp in gen:
-        if comp.text not in BLACKLISTED_COMPLETIONS:
-            completions.append(comp)
+        completions.append(comp)
```
Comment (troydai) [inline src/command_modules/azure-cli-interactive/azclishell/az_completer.py]: You can simply do `completions = list(gen)`

**Comment by derekbekoe** [comment]
Doesn't this conflict with the other PR where you prevented the user from starting 'interactive' from inside interactive mode?

File: src/command_modules/azure-cli-interactive/azclishell/az_completer.py (hunk 1 above)
Comment (williexu) [inline src/command_modules/azure-cli-interactive/azclishell/az_completer.py]: you're right

**Comment by williexu** [comment]
it doesn't, since I didn't actually disable the command. I just changed it so it simply prints a message that the user is already in the shell. This was because `interactive` is already in the help and we didn't want to make it confusing for users

File: src/command_modules/azure-cli-interactive/azclishell/az_completer.py [hunk 2]
```diff
@@ -88,11 +86,7 @@ def _get_weight(val):
             priority = ' '  # a space has the lowest ordinance
         return priority + val.text
 
-    completions = []
-    for comp in gen:
-        if comp.text not in BLACKLISTED_COMPLETIONS:
-            completions.append(comp)
-    return sorted(completions, key=_get_weight)
+    return sorted(list(completions_gen), key=_get_weight)
```
Comment (troydai) [inline src/command_modules/azure-cli-interactive/azclishell/az_completer.py]: You don't have to create a list from `completions_gen`. The `sorted` function will create a new list from the iterable.
Reference: https://docs.python.org/3/library/functions.html#sorted

**Pull request merged**